deps/bazel_dep.MODULE.bazel

```diff
@@ -30,15 +30,10 @@ single_version_override(
 )
 
 # Note that this is 'rules_nodejs-core'
-# Keep in sync with build_bazel_rules_nodejs in WORKSPACE.bzlmod
```

Comment:
What's the WORKSPACE plan? Wait for the move to bzlmod this weekend?

Reply:
There is no `rules_nodejs` in `WORKSPACE`. This is the only place we define it.